README.md

```diff
@@ -1,17 +1,26 @@
 # ohsome API
 
-This REST-based API aims to leverage the tools of the [OSHDB-API](https://github.com/GIScience/oshdb) through allowing to access some of its functionalities via HTTP requests.
-The current stable version is [v1.0.0](https://github.com/GIScience/ohsome-api/releases/tag/1.0.0).
+This REST-based API aims to leverage the tools of the [OSHDB API](https://github.com/GIScience/oshdb) through allowing to access some of its functionalities via HTTP requests. The current stable version is [v1.0.0](https://github.com/GIScience/ohsome-api/releases/tag/1.0.0). The code is developed and maintained by [HeiGIT](https://heigit.org/).
```

Comment:
IMHO this very first introduction sentence could be written even more welcoming and maybe a bit less technical. There would be plenty of opportunity below to explain that it is implemented using the oshdb-api for example. Also the code ownership is not that important to be in the very first paragraph. Instead, maybe we could use this prime "screen real estate" to be a very short introduction of what the ohsome API is and what it does? For example:

> The ohsome API is a generic web API for in-depth analysis of OSM data including it's history. It allows to get aggregated statistics about the evolution of OpenStreetMap data itself and about the contributors behind the data. Furthermore, data extraction methods are provided to access the historic development of individual OSM features.

(just a first very quick first draft)

Reply:
> Also the code ownership is not that important to be in the very first paragraph.

I took this idea directly from the readme of the OSHDB here on GitHub. But yeah we can add some more info and divide it into 2 paragraphs.
How about something like this?

> The ohsome API is a generic web API for in-depth analysis of OpenStreetMap (OSM) data with a focus on it's history. It allows to get aggregated statistics about the evolution of OSM data itself and about the contributors behind the data. Furthermore, data extraction methods are provided to access the historic development of individual OSM features.

> The functionalities of the ohsome API can be accessed via HTTP requests. As a basis underneath serves the OSHDB API. The current stable version is v1.0.0. Developed and maintained by HeiGIT.

Reply:
true, it was even me who added it there: https://github.com/GIScience/oshdb/commit/8447a781356d61c3b695b31f65eb4cc6cfbfc80b :sweat_smile: maybe we should adapt the OSHDB readme file at some point as well…

I like your two-paragraph solution. :+1: